commands/Moderation/mute.js

```diff
@@ -0,0 +1,35 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRole } = msg.guild.settings;
+  const value = user.roles.find("name", muteRole);
+  if (value) {
+    user.removeRole(msg.guild.roles.find("name", muteRole));
```

Comment:
Same here and 2 lines below. And use `await` to avoid raceconditions and uncaught promises.